x-pack/plugin/ql/src/test/java/org/elasticsearch/xpack/ql/tree/NodeSubclassTests.java

```diff
@@ -559,7 +559,7 @@ private Object makeMap(Class<? extends Node<?>> toBuildClass, ParameterizedType
 
     private int randomSizeForCollection(Class<? extends Node<?>> toBuildClass) {
         int minCollectionLength = 0;
-        int maxCollectionLength = 10;
+        int maxCollectionLength = 3;
```

Comment:
Why did you reduce this?

Reply:
Good catch. That's a leftover from an attempt to add EqlNodeSubclassTests to EQL. Will revert it.